.gitignore

```diff
@@ -1,2 +1 @@
 node_modules
-hint-report
```

Comment:
Review : Try to remove any unnecessary file. This comment is **[Optional]**

Reason: By Setting up linters correctly,  you will void issues with any of the linters (weblint, stylelint or lighthouse). I think you start the procedure of adding linters to your code afresh and ignore or remove any unnecessary file.